pyqtgraph/examples/utils.py

```diff
@@ -135,3 +137,28 @@
 skiptest = dict([
     ('ProgressDialog', 'ProgressDialog.py'),    # modal dialog
 ])
+
+
+class FrameCounter(QtCore.QObject):
+    sigFpsUpdate = QtCore.Signal(object)
+
+    def __init__(self, interval=1000):
+        super().__init__()
+        self.frame_cnt = 0
+        self.last_update = 0
+        self.interval = interval
+
+    def update(self):
+        self.frame_cnt += 1
+
+        if self.last_update == 0:
+            self.last_update = perf_counter()
+            self.startTimer(self.interval)
```

Comment:
Do you think we should specify the use of a precise timer here? 

http://doc.qt.io/qt-6/qt.html#TimerType-enum

Reply:
No need. A coarse timer is sufficient because the actual time elapsed is calculated from perf_counter.